modules/apps/change-tracking/change-tracking-web/src/main/java/com/liferay/change/tracking/web/internal/portlet/action/DiscardChangesMVCActionCommand.java

```diff
@@ -45,11 +47,15 @@ protected void doProcessAction(
 		_ctCollectionService.discardCTEntry(
 			ctCollectionId, modelClassNameId, modelClassPK);
 
-		String redirect = ParamUtil.getString(actionRequest, "redirect");
-
-		if (Validator.isNotNull(redirect)) {
-			sendRedirect(actionRequest, actionResponse, redirect);
-		}
+		sendRedirect(
+			actionRequest, actionResponse,
+			PortletURLBuilder.create(
+				PortletURLFactoryUtil.create(
+					actionRequest, CTPortletKeys.PUBLICATIONS,
+					PortletRequest.RENDER_PHASE)
+			).setMVCRenderCommandName(
+				"/change_tracking/view_changes"
+			).buildString());
```

Comment:
Does this redirect work without the `ctCollectionId` param?

Reply:
No it didn't work correctly but there weren't any errors, it went straight to the ongoing publications tab